Header header
geometry_msgs/Vector3 nn_input
geometry_msgs/Vector3 nn_output